libdevice/cmake/modules/SYCLLibdevice.cmake

```diff
@@ -88,6 +95,22 @@ function(add_devicelib_obj obj_filename)
   install(FILES ${devicelib-obj-file}
           DESTINATION ${install_dest_lib}
           COMPONENT libsycldevice)
+
+  set(devicelib-obj-file-new-offload ${obj_new_offload_binary_dir}/${obj_filename}.${new-offload-lib-suffix})
```

Comment:
I understand we're generating fat objects because the offload packager isn't set up to take in raw BC files, but probably that is simplier in the long term. so I'm okay with generating fat objects for now but we should have an internal tracker to enhance clang-linker-wrapper to accept raw BC inputs and then we can simplify/remove some stuff.

Reply:
I have been thinking about this. In my experience, I feel that it's a lot cleaner and in sync with community code to handle packaged files instead of bc files. I suppose there are pros and cons, This is something we can discuss further.